src/types/index.ts

```diff
@@ -81,6 +81,11 @@ export type NormalizedOverrideOutput = {
     route: string,
     verb: Verbs,
   ) => string;
+  operationId?: (
+    operation: OperationObject,
+    route: string,
+    verb: Verbs,
+  ) => string;
```

Comment:
Will this help with my issue also: https://github.com/anymaniax/orval/issues/305  ???

Reply:
It could help but I don't think it's the right fix.
In your issue something is wrong, you shouldn't get a function without only `delete`, it should be `deleteAuthorizationAuthorizationId` if I understand correctly 🤔